examples/storm-mqtt-examples/src/main/java/org/apache/storm/mqtt/examples/MqttBrokerPublisher.java

```diff
@@ -15,35 +15,62 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
 package org.apache.storm.mqtt.examples;
 
+import java.util.Random;
 
 import org.apache.activemq.broker.BrokerService;
 import org.apache.storm.mqtt.MqttLogger;
 import org.fusesource.mqtt.client.BlockingConnection;
 import org.fusesource.mqtt.client.MQTT;
 import org.fusesource.mqtt.client.QoS;
+
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import java.util.Random;
-
-public class MqttBrokerPublisher {
-    private static final Logger LOG = LoggerFactory.getLogger(MqttBrokerPublisher.class);
-
+/**
+ * A MQTT example using a Storm topology.
+ */
+public final class MqttBrokerPublisher {
+    /**
```

Comment:
Same here: I guess checkstyle doesn't require fields to be commented, and the name of fields are self-descriptive. Better to revert them.